### PR TITLE
Data preprocessing with words

### PR DESCRIPTION
**Pull request by kirsten-stallings**
Code for feature extraction and model creation. Still needs the actual tutorial part put at least the models aren't poop!

### CODE REVIEW

File: Code_for_Perfect_LR_model.ipynb [hunk 1]
```diff
@@ -0,0 +1,499 @@
...
+   "cell_type": "code",
+   "execution_count": 150,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "for i, row in df.iterrows():\n",
+    "      for writer in all_writers:\n",
+    "            if writer in row[\"Writer\"]:\n",
+    "                df.at[i, f\"Writer_{writer}\"] = 1\n",
+    "               "
```
Comment (jambran) [inline Code_for_Perfect_LR_model.ipynb]: there may be a way to do this without df.iterrows() (slow) and instead using df.apply(function).

File: Code_for_Perfect_LR_model.ipynb [hunk 2]
```diff
@@ -0,0 +1,499 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 153,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "for column in text_columns:\n",
+    "    df.drop([column], axis=1, inplace=True)"
```
Comment (jambran) [inline Code_for_Perfect_LR_model.ipynb]: can you just do
`df.drop([columns], axis=1, inplace=True)`?

File: Code_for_Perfect_LR_model.ipynb [hunk 3]
```diff
@@ -0,0 +1,499 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 120,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "if train_percent + validation_percent +test_percent < .99999:\n",
+    "    raise ValueError (\"train, validation, and test must sum to 1\")\n",
+    "else:\n",
```
Comment (jambran) [inline Code_for_Perfect_LR_model.ipynb]: consider removing the else

File: Code_for_Perfect_LR_model.ipynb [hunk 4]
```diff
@@ -0,0 +1,499 @@
...
+    "    train, validate, test = np.split(df, [int(train_percent*samples), int((train_percent+validation_percent)*samples)])"
```
Comment (jambran) [inline Code_for_Perfect_LR_model.ipynb]: can we make `validate` either be `val` or `validation`? Thought it was a function a few lines farther down

File: Code_for_Perfect_LR_model.ipynb [hunk 5]
```diff
@@ -0,0 +1,499 @@
...
+    "from sklearn.linear_model import LogisticRegression"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 126,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "lR = LogisticRegression()"
```
Comment (jambran) [inline Code_for_Perfect_LR_model.ipynb]: variable names `lR` should be all lower case

File: Code_for_Perfect_LR_model.ipynb [hunk 6]
```diff
@@ -0,0 +1,499 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 131,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import matplotlib.pyplot as plt\n",
+    "import seaborn as sns\n",
```
Comment (jambran) [inline Code_for_Perfect_LR_model.ipynb]: do you use `matplotlib` or `seaborn`...? Maybe in a later commit?
